forte/processors/stave/stave_processor.py

```diff
@@ -0,0 +1,302 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''
+Package Requirements:
+    tornado == 6.1
+    django == 3.2
+    forte
+    *stave (Required in future version)
+
+Required environment variables:
+*All of the variables below will be deprecated in future version.
+    FRONTEND_BUILD_PATH:
+        Absolute path (or relative path from PYTHONPATH)
+        to stave build folder. Example: "stave/build/"
+    ONTOLOGY_PATH:
+        Absolute path to the ontology file.
```

Comment:
With the recent PR, we can access the ontology content through `Resources` of the pipeline, using the `onto_specs_dict` key. We can also access the path, but the content is probably safer to use if we are working in a multi-machine setting.

https://github.com/asyml/forte/pull/437/files?file-filters%5B%5D=.json&file-filters%5B%5D=.py#diff-c01a327f8bf74824241a66984882d288c8131c47cb4262265bff4f06a88c77a2R165